mmv1/products/vmwareengine/Network.yaml

```diff
@@ -0,0 +1,146 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Network'
+min_version: beta
+base_url: 'projects/{{project}}/locations/{{location}}/vmwareEngineNetworks'
+self_link: 'projects/{{project}}/locations/{{location}}/vmwareEngineNetworks/{{name}}'
+create_url: 'projects/{{project}}/locations/{{location}}/vmwareEngineNetworks?vmwareEngineNetworkId={{name}}'
+update_mask: true
+update_verb: :PATCH
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/vmware-engine/docs/reference/rest/v1/projects.locations.vmwareEngineNetworks'
+description: |
+  Provides connectivity for VMware Engine private clouds.
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+
+import_format: ["projects/{{project}}/locations/{{location}}/vmwareEngineNetworks/{{name}}"]
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vmware_engine_network_legacy"
+    pull_external: true
+    min_version: beta
+    primary_resource_id: "vmw-engine-network"
+    vars:
+      proj_name: "n-"
+      proj_id: "n-"
+    test_env_vars:
+      location: :REGION
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "location"
+    immutable: true
+    required: true
+    url_param_only: true
+    description: |
+      The location where the VMwareEngineNetwork should reside.
+  - !ruby/object:Api::Type::String
+    name: "name"
+    immutable: true
+    required: true
+    url_param_only: true
+    description: |
+      The ID of the VMwareEngineNetwork.
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Creation time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Last update time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      User-provided description for this VMware Engine network.
+  - !ruby/object:Api::Type::Array
+    name: 'vpcNetworks'
+    description: |
```

Comment:
We would normally include an additional update test to verify that the resource can be updated in-place. For this resource, it appears that the description is the only field that can currently be changed, but as other fields are added they can be included in the update test.

Reply:
Agreed. Added tests for network update.